### PR TITLE
add capas to support snow infra

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*
We are not building capas yet, but want to get the manifests into the bundle for e2e testing.  This adds a basic build with hardcoded manifests just to get the flow started.

Todo:

- [x] merge https://github.com/aws/eks-anywhere-build-tooling/pull/460
- [x] cdk changes?
- [x] prow jobs - https://github.com/aws/eks-anywhere-prow-jobs/pull/99
- [ ] add to eks-a bundle (can be after)

/hold

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yxinchen** [comment]
We have a discussion on the provider name, and concluded that we want to use `cluster-api-provider-aws-snow`, the same as our repo's name. Could you please change the provider name accordingly in this pr? Thanks.

File: projects/aws/cluster-api-provider-aws-snow/Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+
+REPO=cluster-api-provider-snow
```
Comment (abhay-krishna) [inline projects/aws/cluster-api-provider-aws-snow/Makefile:4]:
```suggestion
REPO=cluster-api-provder-aws-snow
```

File: projects/aws/cluster-api-provider-aws-snow/Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+
+REPO=cluster-api-provder-aws-snow
```
Comment (abhay-krishna) [inline projects/aws/cluster-api-provider-aws-snow/Makefile:4]: my bad
```suggestion
REPO=cluster-api-provider-aws-snow
```

**Pull request merged**